Test/Units/Test.php

```diff
@@ -47,4 +76,99 @@ public function getFaker($locale = 'en_US')
     {
         return Faker\Factory::create($locale);
     }
+
+    /**
+     * Creates a Kernel.
+     *
+     * Available options:
+     *
+     *  * environment
+     *  * debug
+     *
+     * @param array $options An array of options
+     *
+     * @return HttpKernelInterface A HttpKernelInterface instance
+     */
+    protected function createKernel(array $options = array())
+    {
+        if (null === $this->class) {
+            $this->class = $this->getKernelClass();
+        }
+
+        return new $this->class(
+            isset($options['environment']) ? $options['environment'] : 'test',
+            isset($options['debug']) ? $options['debug'] : true
+        );
+    }
+
+    /**
+     * Attempts to guess the kernel location.
+     *
+     * When the Kernel is located, the file is required.
+     *
+     * @throws \RuntimeException
+     *
+     * @return string The Kernel class name
+     */
+    protected function getKernelClass()
+    {
+        $dir = $this->getKernelDirectory();
+
+        $finder = new Finder();
+        $finder->name('*Kernel.php')->depth(0)->in($dir);
+        $results = iterator_to_array($finder);
+        if (!count($results)) {
+            throw new \RuntimeException(sprintf('Impossible to find a Kernel file, override the %1$s::getKernelDirectory() method or %1$s::createKernel() method.', get_class($this)));
+        }
+
+        $file = current($results);
+        $class = $file->getBasename('.php');
+
+        require_once $file;
+
+        return $class;
+    }
+
+    /**
+     * Override this method if needed
+     *
+     * @return string
+     */
+    protected function getKernelDirectory()
+    {
+        $dir = getcwd().'/app';
+        if (!is_dir($dir)) {
+            $dir = dirname($dir);
+        }
+
+        return $dir;
+    }
+
+    /**
+     * return Kernel
+     *
+     * @return Object HttpKernelInterface
+     */
+    public function getKernel()
+    {
+        if(null === $this->kernel) {
+            $this->kernel = $this->createKernel();
+        }
```

Comment:
Please note this change.
Possible compatibility break, but needed to test CommandTestCase (https://github.com/atoum/AtoumBundle/pull/84/files#diff-9e6f77c9afca2890e3678c26918bc152R29)